zerver/tests/test_events.py

```diff
@@ -258,6 +260,8 @@ def match_states(self, state1, state2):
         def normalize(state):
             # type: (Dict[str, Any]) -> None
             state['realm_users'] = {u['email']: u for u in state['realm_users']}
+            for u in state['subscriptions']:
+                u['subscribers'].sort()
```

Comment:
Because, subscribers list wasn't sorted and that was causing test fail.
![screenshot from 2016-06-29 13 06 04](https://cloud.githubusercontent.com/assets/6594924/16444115/14d61fbc-3dfa-11e6-93e8-e3c47abf270a.png)